nixos/modules/services/desktops/pipewire/wireplumber.nix

```diff
@@ -33,6 +35,12 @@ in
     ];
 
     environment.systemPackages = [ cfg.package ];
+
+    environment.etc."wireplumber/main.lua.d/80-nixos.lua".text = if !pwUsedForAudio then ''
```

Comment:
You can use `lib.optionalString` instead of `if !pwUsedForAudio "<something>" else "";`.

Reply:
Thanks, changed that.